src/test/java/org/takes/tk/TkSlf4jTest.java

```diff
@@ -55,4 +68,28 @@ public void logsException() throws IOException {
         new TkSlf4j(new TkFailure(new IOException(""))).act(new RqFake());
     }
 
+    /**
+     * {@link TkSlf4j} can return an empty response body for {@link TkEmpty}.
```

Comment:
@essobedo do you mean: 

"TkSlf4j can return an empty response body for {@link TkEmpty}."

?